### PR TITLE
Export to sbv

### PR DESCRIPTION
**Pull request by ajayyy**
Untested

### CODE REVIEW

File: src/common/caption-parsers/index.ts [hunk 1]
```diff
@@ -132,13 +133,28 @@ const stringifyNekoToSrt = (captionContainer: CaptionDataContainer) => {
...
+      return {
+        start: Math.floor(caption.start),
+        end: Math.floor(caption.end),
+        text: caption.text,
+      };
+    }),
+    { format: "sbv" }
+  );
+};
+
```
Comment (ajayyy) [inline src/common/caption-parsers/index.ts:148]: Should this be moved to `sbv-parser`?

Comment (nopol10) [reply src/common/caption-parsers/index.ts:148]: Yea, should be more suitable there

**Comment by nopol10** [comment]
Hey @ajayyy are you still working on this or is it ready to be reviewed?

**Comment by ajayyy** [comment]
Sorry, I've been busy with other stuff, so I haven't been able to try to get this built to test it. You can ignore this for now and not worry about merge conflicts.

**Pull request closed without merging**